code/_onclick/ai.dm

```diff
@@ -138,36 +138,35 @@
 /obj/machinery/door/airlock/AICtrlClick() // Bolts doors
 	if(emagged)
 		return
+	
 	if(locked)
-		Topic("aiEnable=4", list("aiEnable"="4"), 1)// 1 meaning no window (consistency!)
```

Comment:
lol what are thoooooseeeeeeeee

Reply:
man this shit hasn't been touched since sayu's rework 4 years ago